astropy/io/votable/tests/vo_test.py

```diff
@@ -75,10 +75,10 @@ def _test_regression(tmpdir, _python_based=False, binary_mode=1):
 
     dtypes = [
         (('string test', 'string_test'), '|O8'),
-        (('fixed string test', 'string_test_2'), '|S10'),
+        (('fixed string test', 'string_test_2'), '<U10'),
```

Comment:
Is this type change in the expected results still abide by the VOTable standards? (Asking as a person unfamiliar with those standards. 😬 )